src/Core/src/Handlers/View/ViewHandlerOfT.Windows.cs

```diff
@@ -21,6 +21,10 @@ protected override void SetupContainer()
 				return;
 			}
 
+			// Check if the parent is a ContentPanel and this view is its Content
```

Comment:
I think this is not a proper fix in viewHandlerOfT.Windows -- I just manually check in on my own ... In WrapperView.Windows , UpdateClip method is handled.. in that if height and width less than 0 it returns... so other othings not happened in the after code in update clip method in WrapperView.Windows..... If I wrapped the same contentview inside the Layout such as stacklayout which is working fine.. the same it is returned first in update clip , after that from Arrange of layout it updated the frame... from the MapFrame it calls again, so at the time child.ActualWdith and child.ActualHeight is determined, and works on that scenario ... can you check like this.. by arranging frame in border , so that we can get updated again in clip with proper height and width values ?

Reply:
Actually I debug it manually , while without fix, the Wrapperview.updateclip is updated properly with visual.clip in Wrapperview... What i was observed is when updating the clip in sizechanged method in sample level ... Contentpanel Visual.Clip (border) is updated lastly may cause the confusion with the wrapperview visual.clip .... byt using the child.Layoutupdated changes in wrapperview working fine ... but iam not sure that is proper solution